manifests/config.pp

```diff
@@ -11,33 +11,35 @@
 # Sample Usage:
 #
 class activemq::config (
-  $server_config,
-  $instance,
-  $package,
-  $path = '/etc/activemq/activemq.xml',
-  $server_config_show_diff = 'UNSET',
+  $server_config           = $activemq::server_config_content,
+  $instance                = $activemq::instance,
+  $package                 = $activemq::package,
```

Comment:
I will remove it. 
`require => Package[$package]` can not be used when we install activemq from a `tar`.